packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/README.md

```diff
@@ -570,6 +572,7 @@ For more info please visit [widget_toolkit_pin](https://pub.dev/packages/widget_
 [patrol_pub_lnk]: https://pub.dev/packages/patrol
 [patrol_cli_pub_lnk]: https://pub.dev/packages/patrol_cli
 [patrol_native_integration_lnk]: https://patrol.leancode.pl/getting-started#integrate-with-native-side
+[patrol_readme_lnk]: https://github.com/Prime-Holding/rx_bloc/blob/develop/packages/rx_bloc_cli/mason_templates/bricks/patrol_integration_tests/README.md
```

Comment:
Please follow the convention
- `docs/patrol_integration_tests.md`

Reply:
fixed

Reply:
Please use a relative path. @DDavidPrime is there a need the script that handles the relative paths to be adjusted ?

Reply:
@DimitarStoyanov91 @StanevPrime Yep, the relative path and the full path should be included in the `rx_bloc_cli/bin/compile_test_project.sh` so they work with generated projects and the pub.dev page.

Reply:
fixed